### ISSUE TITLE
Ente [Auth] - Duplicate Auth entries on the Web Interface

### ISSUE THREAD

**Issue by nvgoldendog** [ente-io/ente#4274]
### Discussed in https://github.com/ente-io/ente/discussions/4272

<div type='discussions-op-text'>

<sup>Originally posted by **nvgoldendog** December  1, 2024</sup>
Hi. I was migrating from Authy to Ente Auth and created a TEXT file with a single line of the otpauth per the instructions like:
otpauth://totp/SOMEWHERE?secret=112P4C5RVQB2RVGCYKJDCJKOYY&digits=6&period=30
On each line. I then launched the Windows application, had previously created the account with the photo side of Ente not realizing there were two basically different products, signed in to it, did an import and then noticed that I had several entries that were duplicated. I compared the name, the CAPS and not, spacing, the actual secret, all the same, but only once in my import file not listed twice which is what I expected a  mistake by me. I then installed the Android application and again it produced duplicates that were the same as the Windows Application. So I thought the best thing to do would be do delete the duplicates and I did.  Once deleted I was fine on the Windows Application and Android, however on the website they are still listed twice. Is it because I didn't follow the exact syntax  of **otpauth"//totp/SOMEWHERE/com:myemail@domain.com?secret=112P4C5RVQB2RVGCYKJDCJKOYY** and I do not have the email address in my export and have the output length specified as well and the time period?  I opened one of the duplicates on the Windows Application and added my email address on the accounts line and saved it, logged out of the web app, reopened it, and now I see my email address on the one I edited but on the other duplicate I do not see the email address changed nor it just gone like it is on the Android application and the Windows Application,  Is this a bug I hit or am I not following the instructions which seemed well documented and I understand very well.  Web version:
![image](https://github.com/user-attachments/assets/bec8586d-885e-41ed-a21e-4670d253606c)

Windows Version: 
![image](https://github.com/user-attachments/assets/eb95e90e-c1b3-415f-8f20-c3d59c37e4fa)
</div>

**Comment by mnvr** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/4272
